workers/unity/Assets/Playground/Scripts/Spawning/ArchetypeInitializationSystem.cs

```diff
@@ -48,7 +48,9 @@ protected override void OnCreateManager(int capacity)
 
             if (!(worker is UnityClient) && !(worker is UnityGameLogic))
             {
-                Debug.LogErrorFormat(Errors.UnknownWorkerType, World.Name);
+                view.LogDispatcher.HandleLog(new LogEvent("Unknown worker type.", LogType.Error)
```

Comment:
Same comment as before. The message should tell the user everything they need for how to fix this. So ideally there should be some context in this. In this case you can actually say exactly what has gone wrong and how to fix it.